iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/XMLExportOptions.java

```diff
@@ -24,14 +27,17 @@
 
     private boolean skipExtensions = false;
 
+    private TopologyLevel topologyLevel = TopologyLevel.NODE_BREAKER;
+
     public XMLExportOptions() {
     }
 
-    public XMLExportOptions(boolean withBranchSV, boolean forceBusBranchTopo, boolean indent, boolean onlyMainCc) {
+    public XMLExportOptions(boolean withBranchSV, boolean forceBusBranchTopo, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel) {
```

Comment:
You have to create 2 different constructors: you mussn't change the signature of the old one (backward compatibility). Create a new one:
```
public XMLExportOptions(boolean forceBusBranchTopo, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel)
```